### PR TITLE
About page

### PR DESCRIPTION
**Pull request by 321jellyfish**
I have put some comments in the Trello to explain some of my decisions. 

Note: I have only one module styling sheet - but I have extracting the sections out into components for ease of reading - they are still all using one module styling sheet to avoid unnecessary repetition.

### CODE REVIEW

File: frontend/src/app/about/FutureFunctionality.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    return (
+        <section className={styles.section}>
+          <h2 className={styles.subheading}>Future functionality</h2>
+          <p>
+            In the future we would like to add further functionality to the
+            website.
+          </p>
+          <p>Including the following features:</p>
+          <ul className={styles.futures_features_list}>
+            <li>Users can log in</li>
```
Comment (jesswils) [inline frontend/src/app/about/FutureFunctionality.tsx]: Forgive me if I’ve missed this but with the user being able to login, would that mean they would have their own profile? If so I would say something like  “Add authentication and user profile”

File: frontend/src/app/about/ProjectInfo.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+          <h2 className={styles.subheading}>About the project</h2>
+          <p>
+            Postbox library was inspired by our love of books. The aim of the
+            website is to make it easy to share books with your friends. Users
+            list books they want to lend and other users can request to borrow
+            those books - sharing the literary love!{' '}
+          </p>
+          <p>
+            We built this website in late 2023 and early 2024. Our aim was to a
+            build a full stack web app, in order to deepen our knowledge of
```
Comment (jesswils) [inline frontend/src/app/about/ProjectInfo.tsx:14]: Wee typo “our aim was to build”

File: frontend/src/app/about/ProjectInfo.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+            working with data, UX design and the latest version of Next.js and
+            the App Router. We hope you enjoy your time at Postbox Library.
+          </p>
+          <p>We hope you enjoy your time at Postbox Library.</p>
```
Comment (jesswils) [inline frontend/src/app/about/ProjectInfo.tsx:18]: Duplicate

File: frontend/src/app/about/page.module.css [hunk 4]
```diff
@@ -3,22 +3,34 @@
...
 }
 
 .subheading {
-    color: #f472b6;
+    color: hotpink;
     margin: 10px;
     text-align: center;
 }
 
 .profile_photo {
```
Comment (nolixul) [inline frontend/src/app/about/page.module.css:31]: Could: The profile photos have rounded corners or be circles?
Could you also try the photos as a smaller size and see if it makes the page looks better, as there's not much content next to them? :)

File: frontend/src/app/about/page.module.css [hunk 5]
```diff
@@ -3,22 +3,34 @@
     margin: 20px;
 }
 
+.container {
```
Comment (nolixul) [inline frontend/src/app/about/page.module.css:6]: Could: The container have a little more left and right padding so it looks a bit better in mobile view?

File: frontend/src/app/about/Creators.tsx [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+export default function Creators () {
+    return (
+        <section className={styles.about_the_creators_container}>
+          <h2
+            className={`${styles.subheading} ${styles.about_the_creators_header}`}>
+            About the creators
+          </h2>
+
+          <div className={styles.creator_text_container}>
+            <h3 className={styles.creator_name}>Charlotte</h3>
```
Comment (nolixul) [inline frontend/src/app/about/Creators.tsx:16]: Should: The creator_names should be aligned properly with the text underneath

**Review by nolixul — COMMENTED** [review]
It looks really good :) I'll take another pass at it later but it looks in great shape <3

File: frontend/src/app/about/FutureFunctionality.tsx (hunk 1 above)
Comment (321jellyfish) [reply frontend/src/app/about/FutureFunctionality.tsx]: Updated to new wording

File: frontend/src/app/about/ProjectInfo.tsx (hunk 2 above)
Comment (321jellyfish) [reply frontend/src/app/about/ProjectInfo.tsx:14]: Updated

File: frontend/src/app/about/ProjectInfo.tsx (hunk 3 above)
Comment (321jellyfish) [reply frontend/src/app/about/ProjectInfo.tsx:18]: Removed, good spot

File: frontend/src/app/about/page.module.css (hunk 4 above)
Comment (321jellyfish) [reply frontend/src/app/about/page.module.css:31]: Have made smaller and circular

File: frontend/src/app/about/Creators.tsx (hunk 6 above)
Comment (321jellyfish) [reply frontend/src/app/about/Creators.tsx:16]: Done 👍

File: frontend/src/app/about/page.module.css (hunk 5 above)
Comment (321jellyfish) [reply frontend/src/app/about/page.module.css:6]: I think it looks nice with the padding as it is. Can we keep it with this? then when it's deployed to Vercel I will check it on my phone and see how it looks?

**Comment by jesswils** [comment]
Could you add the body font in global styling please? :)

**Comment by 321jellyfish** [comment]
@jesswils what do you mean? font-family: var(--font-body) is set on the html in the global.css? What do you want me to add?

**Comment by jesswils** [comment]
> @jesswils what do you mean? font-family: var(--font-body) is set on the html in the global.css? What do you want me to add?

Oh how odd. When I checked out the code the body font wasn’t showing? It was all heading font for me. I’ll check it again.

**Review by nolixul — APPROVED** [review]

**Pull request merged**